src/components/ids-swipe-action/ids-swipe-action.js

```diff
@@ -63,7 +63,9 @@ class IdsSwipeAction extends mix(IdsElement).with(IdsEventsMixin, IdsThemeMixin)
     }
 
     if (this.leftButton && this.swipeType === 'reveal') {
-      this.container.scrollLeft = 85;
+      setTimeout(() => {
```

Comment:
We're trying to standardize these in favor of `requestAnimationFrame()`:

```suggestion
      requestAnimationFrame(() => {
```

Reply:
@EdwardCoyle 
I tried with requestAnimationFrame but it was same. What can I do for now?

Reply:
It seems to work with it for me? The only thing i see is sometimes i get an extra line on the border:

![Screen Shot 2021-12-01 at 12 54 08 PM](https://user-images.githubusercontent.com/814283/144287521-de0ecff4-1a34-4085-9d7a-7721cb56277b.png)

Reply:
You don't see any swipe left animation on page load?

Reply:
I see it move across the page (which isnt ideal) but i see the same thing with setTimeout or requestAnimationFrame

Reply:
Yes, it's same. The purpose was to remove that animation on the page load.